pages/api/request/help.ts

```diff
@@ -0,0 +1,88 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { cors, middleware, sessionExists } from "@lib/middleware";
+import { NotifyClient } from "notifications-node-client";
+import { logMessage } from "@lib/logger";
+import { MiddlewareProps } from "@lib/types";
+
+const SUPPORT_EMAIL_ADDRESS = "assistance+forms@cds-snc.freshdesk.com";
+const CONTACTUS_EMAIL_ADDRESS = "jose.jimenez@cds-snc.ca";
+
+const requestSupport = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+  { session }: MiddlewareProps
+) => {
+  try {
+    if (!session) return res.status(403).json({});
+
+    const { type, name, email, request, description } = req.body;
+
+    if (!name || !email || !request || !description) {
+      return res.status(404).json({ error: "Malformed request" });
+    }
+
+    const templateID = process.env.TEMPLATE_ID;
+    const notifyClient = new NotifyClient(
+      "https://api.notification.canada.ca",
+      process.env.NOTIFY_API_KEY
+    );
+
+    const TO_EMAIL_ADDRESS = type === "support" ? SUPPORT_EMAIL_ADDRESS : CONTACTUS_EMAIL_ADDRESS;
+
+    // Here is the documentation for the `sendEmail` function: https://docs.notifications.service.gov.uk/node.html#send-an-email
+    await notifyClient.sendEmail(templateID, TO_EMAIL_ADDRESS, {
+      personalisation: {
+        subject:
+          type === "support"
+            ? "Support request / Demande de soutien"
+            : "Contact request / Demande de soutien",
+        formResponse:
+          type === "support"
```

Comment:
This seems very fragile and difficult to read. It might be worth to separate the two cases as if statements to increase readability

Reply:
If this was turned into a util it would make it easier 
```
sendEmail(to, subject, message)
```

Reply:
I agree it's not pretty :) Sure a send email utility sounds good

Reply:
Think thats probably the better idea. Follow @timarney advice